src/components/avatars.css

```diff
@@ -0,0 +1,39 @@
+.avatar {
+  width: 80px;
+  height: 80px;
+  border-radius: var(--radius-round);
+  border: 4px solid var(--black-inverted);
+  box-shadow: 0px 0px 5px var(--black-300);
+  box-shadow: var(--shadow-smooth);
+  background-size: contain;
+}
+
+.avatar-text {
+  width: 80px;
+  height: 80px;
+  border-radius: var(--radius-round);
+  border: 4px solid var(--black-inverted);
+  box-shadow: 0px 0px 5px var(--black-300);
+  font-size: var(--tx-24);
+  font-weight: var(--tx-bold);
+  display: flex;
```

Comment:
Can move this flex properties to `utilities` and directly use the class on element, will be reusable for other components as well